openwisp_monitoring/check/tasks.py

```diff
@@ -57,17 +65,19 @@ def auto_create_ping(model, app_label, object_id):
     if has_check:
         return
     ct = ContentType.objects.get(app_label=app_label, model=model)
```

Comment:
can you load ContentType with the migration framework as well please?
And test it again.

Reply:
It worked, thanks :+1: